lib/livebook_web/live/output/control_component.ex

```diff
@@ -13,6 +13,8 @@ defmodule LivebookWeb.Output.ControlComponent do
       class="flex"
       id={"#{@id}-root"}
       phx-hook="KeyboardControl"
+      data-cell-id={@cell_id}
+      data-default-handlers={to_string(Map.get(@attrs, :default_handlers, :off))}
```

Comment:
```suggestion
      data-default-handlers={Map.get(@attrs, :default_handlers, :off)}
```

We have to_string below because they are booleans. :)